src/games/strategy/triplea/ai/proAI/ProBidAI.java

```diff
@@ -1968,24 +1976,24 @@ private static Route getMaxSeaRoute(final GameData data, final Territory start,
   }
 
   private static void reorder(final List<?> reorder, final Map<?, ? extends Number> map, final boolean greaterThan) {
```

Comment:
@RoiEXLab  I agree, but the change here is simply to reformat the code. Ideally that would have been done as a different commit to allow for an easy diff of the changes that need review.

@ron-murhammer needing to reformat an entire class should be a rare event. If reformat is necessary, then let's try to do it on the entire project when we do. In theory the change set should be minimal since we have already run the formatter on the entire project. Failing that,  please make the reformat changes in different commit, so we can skip review of formatting changes.